Makefile

```diff
@@ -199,3 +202,7 @@ fmt:
 .PHONY: fmtcheck
 fmtcheck:
 	@gofmt -l -s $(SOURCE_DIRS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
+
+.PHONY: validate_commits
+validate_commits: $(GOPATH)/bin/git-validation
+	git-validation -q -run short-subject,message_regexp='(Issue #[0-9]+ [A-Z].*|cut v[0-9]+.[0-9]+.[0-9]+)' -range master..HEAD
```

Comment:
Do we really need to be so pedantic to disallow a commit when the message starts with a lower case ? Come on...

Reply:
Can be changed ;-)

Reply:
I see we need the option to specify like: "Issue #xxx #xxx", so "Issue #[0-9]+" would be enough I guess.